Xwt/Xwt/MessageDialog.cs

```diff
@@ -49,10 +49,20 @@ public static void ShowError (string primaryText, string secondaryText)
 		}
 		public static void ShowError (WindowFrame parent, string primaryText, string secondaryText)
 		{
-			GenericAlert (parent, Toolkit.CurrentEngine.Defaults.MessageDialog.ErrorIcon, primaryText, secondaryText, Command.Ok);
+			ShowError(parent, primaryText, secondaryText, null);
+		}
+		public static void ShowError(string primaryText, string secondaryText, string title)
+		{
+			ShowError(RootWindow, primaryText, secondaryText, title);
+		}
+		public static void ShowError(WindowFrame parent, string primaryText, string secondaryText, string title)
+		{
+			if (string.IsNullOrEmpty(title))
+				title = (parent != null) ? parent.Title ?? "Error" : "Error";
```

Comment:
This can be simplified and we need translation support:
```suggestion
				title = parent?.Title ?? Application.TranslationCatalog.GetString ("Error");
```